serve_test.go

```diff
@@ -268,14 +268,14 @@ func TestServeCacheNegotiatingContent(t *testing.T) {
 		logOutput.Reset()
 	})
 
-	t.Run("immediately re-request but different accept header, cache should be cold", func(t *testing.T) {
+	t.Run("immediately re-request but different accept header, cache should be warm", func(t *testing.T) {
```

Comment:
🤕 on my part!

Reply:
Nah, those tests were great for implementing this feature. Made my life so much easier.